c/example-1-hello-world/README.md

```diff
@@ -0,0 +1,49 @@
+# Hello, world!
+
+This is the simplest contract in the repository, it adds 2 and 2 and 
```

Comment:
This contract adds 2 and 2 and 
returns the sum as an external message. Since parsing external messages 
is not yet supported, this contract is providing for testing purposes only.